### PR TITLE
feat: error tracking group view page

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Add a view page for error groups

## Changes

Difficult to screenshot given I have no `$exception` events locally

Using `$exception_type` as the most rudimentary grouping until we have a better algo

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Replaced the `$exception` events with `$pageview`

### CODE REVIEW

File: frontend/src/scenes/error-tracking/ErrorTrackingGroupScene.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    paramsToProps: ({ params: { id } }): (typeof errorTrackingGroupSceneLogic)['props'] => ({
+        id,
+    }),
+}
+
+export function ErrorTrackingGroupScene(): JSX.Element {
+    const { eventProperties } = useValues(errorTrackingGroupSceneLogic)
+
+    return eventProperties && eventProperties.length > 0 ? (
+        <ErrorDisplay eventProperties={eventProperties[0]} />
```
Comment (daibhin) [inline frontend/src/scenes/error-tracking/ErrorTrackingGroupScene.tsx:20]: Just showing a single one. Will come back and build a paginator

**Comment by marandaneto** [comment]
What's the difference between the error tracking and error tracking group? a bit confused.
I think it's related to "list all groups", then you click on one of them, and list all exceptions that belong to that group. If that's the case, the error tracking page should not be part of the lateral menu, but only the error tracking group page, unless you want to build something similar to the "Activity" page for events, but I'd say it less useful since the aggregated results delivery more value.

**Comment by pauldambra** [comment]
> the error tracking page should not be part of the lateral menu, but only the error tracking group page, unless you want to build something similar to the "Activity" page for exceptions,

yep, ultimately, individual exceptions are only examples for grouped exceptions

**Comment by marandaneto** [comment]
@daibhin 

> Using $exception_type as the most rudimentary grouping until we have a better algo

keep in mind that an exception have causes (nested exceptions) in some languages such as JS, Java, C#, so consider that 1 exception may contain more exceptions in it

eg https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause

**Comment by daibhin** [comment]
@marandaneto correct, there is a split between a group list & individual group scene:

- **ErrorTrackingScene:** Shows all the error groups in a table
- **ErrorTrackingGroupScene:** Linked to from the `ErrorTrackingScene` table. Shows individual errors in the group (and in future replays)

Right now only the `ErrorTrackingScene` is in the left hand nav as the entry screen to the product. No activity like scene showing individual errors

-----

> keep in mind that an exception have causes (nested exceptions) in some languages such as JS, Java, C#, so consider that 1 exception may contain more exceptions in it

TIL. I'm not even sure we support that in the capture of exceptions so will bear it in mind going forward but probably ok to ignore given the `$exception_type` grouping is really just a stop gap while we get the embeddings / fingerprinting in place

**Review by marandaneto — APPROVED** [review]

**Pull request merged**